src/components/ProductList.jsx

```
@@ -0,0 +1,98 @@
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { ArrowGlyph, Eyebrow, StackRow, Tag } from './Editorial';
+
+export default function ProductList({ products, startIndex = 1 }) {
```

ESLint

[error] 'products' is missing in props validation

(react/prop-types)